### ISSUE TITLE
Add icon for linux

### ISSUE THREAD

**Issue by pl0q1n** [0xd34d10cc/shar#258]
After #257 we need to provide icon for Linux, and it's not that easy. Useful links for future me/you:
https://linuxcritic.wordpress.com/2010/04/07/anatomy-of-a-desktop-file/
https://github.com/albertlauncher/albert/issues/505

**Comment by 0xd34d10cc** [comment]
Closing in favor of #185
